src/app/user/(logged-in)/my-account/page.tsx

```diff
@@ -119,14 +120,16 @@ export default async function Page() {
         <p className="text-sm font-bold">User ID Number</p>
         <p>{actor.getUserId()}</p>
       </div>
-      <div className="flex flex-col gap-3">
-        <BarkButton
-          className="w-full md:w-32"
-          variant="brand"
-          href={RoutePath.USER_MY_ACCOUNT_EDIT}
+      <div className="flex gap-3">
+        <Link
+          className={`h-[60px] ${buttonVariants({ variant: "brand" })}`}
+          href={{
+            pathname: RoutePath.USER_MY_ACCOUNT_EDIT,
+            query: { userName, userPhoneNumber, userResidency, userEmail },
+          }}
```

Comment:
1: My w-full button! 😭  — I recently changed this to a `w-full` button for mobile screens following this guide - https://uxdesign.cc/where-to-put-the-primary-button-5bd8cb1764c4

2: The /user/my-account/edit page should load its own data. It shouldn't rely on query params from the caller.

Reply:
1. Done 8261d25be503f878d3e220196caea933fedbdd47 
- Haha I think I explicitly set it this way due to the previously existing Delete button and nudging them into place to fit Figma

2. Done 8c5e8b266dce1f57fcb31f1545551c3fc6297b30